.buildkite/pipeline_pr.py

```diff
@@ -49,7 +49,7 @@
 
 release_grp = group(
     "📦 Release Sanity Build",
-    "./tools/devtool -y sh ./tools/release.sh --libc musl --profile release --make-release",
+    "touch ./tests/test-report.json && ./tools/devtool -y sh ./tools/release.sh --libc musl --profile release --make-release",
```

Comment:
Mhh, why? It should normally put this file into `./test_results/test-report.json` :thinking:

Reply:
Good catch.

Yep, we have to remove the option from here and fix the release.sh to match the new results location: https://github.com/firecracker-microvm/firecracker/blob/main/tools/devtool#L486

I forgot to do it during 41412c14d5d2697a95e121869d3d8c4352087564. @sudanl0 could you include that fix in this PR?